R/max_gap.R

```diff
@@ -118,11 +118,18 @@ maximal_meaningful_interval <- function(x) {
 #' }
 #' @export
 find_all_meaningful_gap <- function(x, change_points) {
+  UseMethod('find_all_meaningful_gap')
+}
+
+#' @exportS3Method find_all_meaningful_gap Histogram
+find_all_meaningful_gap.Histogram <- function(x, change_points){
+  find_all_meaningful_gap.numeric(x = x$histogram_data, change_points = change_points)
+}
 
+#' @exportS3Method find_all_meaningful_gap numeric
+find_all_meaningful_gap.numeric <- function(x, change_points){
+  
```

Comment:
Use `@rdname find_all_meaningful_gap ` to link all pages together. See other comment with more info.